tests/testthat/test-rng.R

```diff
@@ -64,6 +64,16 @@ test_that("run uniform random numbers with odd bounds", {
 })
 
 
+test_that("distribution of uniform numbers", {
```

Comment:
Do we have an `rnorm` test too?

Reply:
there is one:

```r
test_that("norm_rand agrees with rnorm", {
  n <- 100000
  ans <- dust_rng$new(2, 1)$norm_rand(n)
  expect_equal(mean(ans), 0, tolerance = 1e-2)
  expect_equal(var(ans), 1, tolerance = 1e-2)
  expect_gt(ks.test(ans, "pnorm")$p.value, 0.1)
})
```

but I can add one that changes parameters too